src/ext/oblivc/obliv_bits.c

```diff
@@ -56,7 +56,6 @@ static struct stdioTransport stdioTransport
 void protocolUseStdio(ProtocolDesc* pd)
   { pd->trans = &stdioTransport.cb; }
 
-//#define PROFILE_NETWORK
```

Comment:
Well, I ran the benchmarks, and stand corrected. The difference is (very slightly) noticeable, at least for CPU-constrained tasks. Results are attached. I will try putting all of the profiling stuff behind a runtime option and see what happens. If I'm going to go to the effort, should I also move gate counting behind an option as well? It stands to reason that if the counting the bytes degrades performance, then maybe counting gates does too.

[benchmarks.zip](https://github.com/samee/obliv-c/files/743203/benchmarks.zip)